### ISSUE TITLE
Clarify return values for JSON-RPC methods miner_start and miner_stop

### ISSUE THREAD

**Issue by MadelineMurray** [hyperledger/besu-docs#251]
Some discussion around the desired return values from JSON-RPC methods miner_start and miner_stop came up in the following PR:
https://github.com/hyperledger/besu/pull/104#discussion_r334708045

As of #104, miner_start will return true if the node is already mining or mining was successfully started. And miner_stop will return true if the node was already idle or mining was stopped.

An alternative would be to return true if the mining status was actually updated (similar to the behavior of other methods like admin_addPeer / admin_removePeer which return true if the node was successfully modified).

We should decide on the desired behavior and update our documentation to clarify the meaning of the returned boolean: https://besu.hyperledger.org/en/latest/Reference/API-Methods/#miner_start

---------------------------------------

We will keep the current functionality, but we should update the documentation to be more explicit.

In short
￼miner_start￼: returns ￼true￼ if mining was started or if the node was already mining
￼miner_stop￼: returns ￼true￼ if mining was stopped or if the node was not mining
